src/main/include/log4cxx/log4cxx.h.in

```diff
@@ -64,8 +64,15 @@ typedef unsigned int log4cxx_uint32_t;
 	typedef std::weak_ptr<T> T##WeakPtr
 #define LOG4CXX_UNIQUE_PTR_DEF(T) typedef std::unique_ptr<T> T##UniquePtr;
 #define LOG4CXX_LIST_DEF(N, T) typedef std::vector<T> N
+#define LOG4CXX_PRIVATE_PTR(T) std::unique_ptr<T>
```

Comment:
Is this define really needed?

Reply:
I only wanted the definitions to be in the same place, so the usage is (for example)
		LOG4CXX_DECLARE_PRIVATE_MEMBER_PTR(AppenderSkeletonPrivate, m_priv)
		AppenderSkeleton(LOG4CXX_PRIVATE_PTR(AppenderSkeletonPrivate) priv);

Reply:
It feels like it doesn't really add anything in terms of making things simpler to read(to me).

Not really a huge issue, more of a preference thing in how to organize the code.